<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleUnitInfo</class>
 <widget class="QWidget" name="SingleUnitInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>新建 Unit...</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="infoBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name">
          <property name="styleSheet">
           <string notr="true">font: bold 16pt &quot;宋体&quot;;</string>
          </property>
          <property name="text">
           <string>名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="description">
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;仿宋&quot;;</string>
          </property>
          <property name="text">
           <string>描述</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hpLabel">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>血量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mpLabel">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>移动力</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ceLabel">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>攻击力</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="newButton">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;宋体&quot;;</string>
          </property>
          <property name="text">
           <string>新建！</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
